packages/kit/test/apps/amp/package.json

```diff
@@ -20,7 +20,7 @@
 		"dropcss": "^1.0.16",
 		"svelte": "^5.2.9",
 		"svelte-check": "^4.1.1",
-		"typescript": "^5.5.4",
```

Comment:
I think the test apps were using TS 5.5.4 because it was needed when we upgraded them to use Svelte 5. Running `pnpm check` doesn't reveal the type errors now because I think the pnpm-lock file is using TS 5.7.3.

Reply:
Ah, I saw 5.5.3 some places and 5.5.4 some places. Perhaps we should standardize on 5.5.4 then